core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/provider_life_cycle.go

```diff
@@ -129,11 +129,11 @@ func (p *logEventProvider) UnregisterFilter(upkeepID *big.Int) error {
 
 // newLogFilter creates logpoller.Filter from the given upkeep config
 func (p *logEventProvider) newLogFilter(upkeepID *big.Int, cfg LogTriggerConfig) logpoller.Filter {
-	topics := p.getFiltersBySelector(cfg.FilterSelector, cfg.Topic1[:], cfg.Topic2[:], cfg.Topic3[:])
```

Comment:
p.getFiltersBySelector can likely be deleted here

Reply:
this is already deleted